syncer/ddl.go

```diff
@@ -91,7 +91,7 @@ func (s *Syncer) parseDDLSQL(sql string, p *parser.Parser, schema string) (resul
 			if len(schema2) > 0 {
 				schema = schema2
 			}
-			ignore, err2 := s.skipDMLEvent(schema, table, replication.QUERY_EVENT)
+			ignore, err2 := s.skipDMLEvent(&filter.Table{Schema: schema, Name: table}, replication.QUERY_EVENT)
```

Comment:
maybe func`tableNameForDML ` can directly return `filter.Table`

Reply:
SGTM. This pr is very huge. Maybe changing it next pr is better.